src/elements/tag/tag/tag.ts

```diff
@@ -27,6 +37,9 @@ export type SbbTagSize = 's' | 'm';
 export
 @customElement('sbb-tag')
 @slotState()
+@hostAttributes({
```

Comment:
Does it not work when setting aria-pressed always together with data-checked? I'd appreciate one single point of toggling